core/services/blockhashstore/feeder.go

```diff
@@ -0,0 +1,156 @@
+package blockhashstore
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink/core/logger"
+)
+
+// Event contains metadata about a VRF randomness request or fulfillment.
+type Event struct {
+	// ID of the relevant VRF request.
+	ID string
```

Comment:
Might be worth noting here that for V1 the request ID is a bytes32 (so I'm guessing it'll show up here as 0x... when encoded as a string) and for V2 the request ID is a uint256, which is a `*big.Int` in Go and will show up as a regular integer in the string.